packages/cli/src/commands/generate/dbAuth/templates/login.tsx.template

```diff
@@ -21,13 +21,13 @@ const LoginPage = () => {
     }
   }, [isAuthenticated])
 
-  const usernameRef = useRef()
+  const usernameRef = useRef<HTMLInputElement>(null)
   useEffect(() => {
     usernameRef.current.focus()
   }, [])
 
   const onSubmit = async (data) => {
-    const response = await logIn({ ...data })
+    const response: any = await logIn({ ...data })
```

Comment:
Just a quick fix, the decent fix should be proper type the logIn method, but does this `logIn` being used for all scenarios and some of them do return while some of them are not? That gonna make it more interesting...

Reply:
I don't think we need to type the response here, the type can probably be inferred from the login method - which I suppose currently is `any`.

Reply:
You are right, should be inferred, the current type is `void` which causes the problem.

this `any` is just for removing that annoying red underline in the IDE xD,

Reply:
Gotcha - let me check the typings for useAuth :)

Reply:
Albert, what do you think of modifying the return type in https://github.com/redwoodjs/redwood/blob/main/packages/auth/src/AuthProvider.tsx to be `Promise<any>`? 

I have plans to improve these typings, but will have to wait a little

Reply:
good idea, that would be a better fix since this is the source. Promise<any> would be better because unknown means the we have to cast it when use it.

We can just let that function return Promise<any> and properly type it when the API design is finalized.